docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# CONTRIBUTING GUIDE
+
+## Pull Request
+
+Before you submit a pull request please adhere to the following guidelines:
+
+create branch using the following convention:\
+`<type>/<name>`
+
+There are following types of branches:
+
+- `feat` for branches that introduce a new feature to the codebase
+- `fix` for branches that patch a bug in the codebase
+- `test` for branches that add missing tests or correcting existing tests
+- `docs` for branches that affect documentation only
+- `refactor` for branches with code change that neither fixes a bug nor adds a feature
+- `style` for branches that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- `build` for branches that affect build components like build tool, ci pipeline, dependencies
```

Comment:
I think 'ci' should be in capital letters

Reply:
resolved